content/preferences-settings/general.md

```diff
@@ -27,12 +27,16 @@ GUI controls and text DPI
 
 # CSS Theme Modifications
 
-In addition to selecting a pre-built theme you can also apply additional CSS customisations of your own to tweak the look-and-feel of darktable. A text box is provided for this purpose.
+In addition to selecting a pre-built theme you can also apply additional CSS customisations of your own to tweak the look-and-feel of darktable. darktable provides a textbox for this purpose in general tab of main preferences. No need to set import lines of darktable CSS themes. You just need to choose which default darktable theme you want to start from (in theme menu just above) and then just copy/paste tweaks you find or create your own.
```

Comment:
You don't need to mention the "general tab of main preferences" since this entire section of the user manual is _about_ the general tab of preferences.

Also, we shouldn't start sentences with lowercase letters. In order to not annoy the "darktable always has a lower case d" people, I therefore avoid at all costs putting "darktable" at the start of a sentence (hence the previous wording).